src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp

```diff
@@ -252,8 +252,11 @@ class KerrSchild : public AnalyticSolution<3_st>,
       const tnsr::I<DataType, volume_dim, Frame>& x, double /*t*/,
       tmpl::list<Tags...> /*meta*/) const {
     static_assert(
-        tmpl2::flat_all_v<
-            tmpl::list_contains_v<tags<DataType, Frame>, Tags>...>,
+        tmpl2::flat_all_v<tmpl::list_contains_v<
```

Comment:
can you make sure the KS test doesn't retrieve via the intermediate vars?

Reply:
I'm not sure I follow :/ [Here](https://github.com/sxs-collaboration/spectre/blob/7b8e2062f46dd6e71331b5930511f58aef2cd8c1/tests/Unit/Evolution/Systems/Ccz4/Test_Ricci.cpp#L79) it's retrieved without using intermediate vars, but my understanding is that `KerrSchild` will still use it internally. Let me know if you think I'm misunderstanding your request!

Reply:
Sorry, the call https://github.com/sxs-collaboration/spectre/blob/7b8e2062f46dd6e71331b5930511f58aef2cd8c1/tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_KerrSchild.cpp#L208 should not go through the intermediate vars is what I'm saying. Does that clarify things?

Reply:
Oh I see, I definitely misunderstood. I will make that fix now. Thanks for clarifying!

Reply:
@nilsdeppe I've squashed in this fix